app/templates/Default/about.latte

```diff
@@ -4,31 +4,31 @@
 <h1 class="">O projektu</h1>
 
 <div class="row">
-    <div class="col-sm-9">
+    <div class="col-12">
         <p>
             Skautské hospodaření vzniklo na základě potřeby systematizovat vyúčtování víkendových výprav a tím minimalizovat počty chyb a nejasností s tím spojených.
             Projekt vznikl v roce 2009 a od té doby je průběžne rozšiřován podle možností a potřeby.
         </p>
         <p>
             Na podzim roku 2016 se k rozvoji projektu připojil František Maša a díky tomu získal projekt více nových funkcí a profesionálnější programátorské zázemí.
         </p>
-        <h2 id="ucel">K čemu slouží?</h2>
+        <h2 id="ucel"><a href="#ucel">K čemu slouží?</a></h2>
```

Comment:
Použil bych radši [anchorjs](https://www.bryanbraun.com/anchorjs) pro přidání kotev ([ukázky](https://www.bryanbraun.com/anchorjs/#examples)). Vypadá to IMO líp, než když má nadpis stejnej styl jako odkaz.